openedx/core/djangoapps/programs/tests/test_signals.py

```diff
@@ -75,3 +76,65 @@ def test_programs_enabled(self, mock_is_learner_issuance_enabled, mock_task):
         self.assertEqual(mock_is_learner_issuance_enabled.call_count, 1)
         self.assertEqual(mock_task.call_count, 1)
         self.assertEqual(mock_task.call_args[0], (TEST_USERNAME,))
+
+
+@attr(shard=2)
```

Comment:
What does that do?

Reply:
My understanding is it determines which batch of code this runs in, so you'll see these tests run on shard 2 always.